### PR TITLE
CustomBallot and MyToken contracts annotated using NatSpec.

### LINKED ISSUE #14: Document contracts using NatSpec
It is recommended that Solidity contracts are fully annotated using [NatSpec](https://docs.soliditylang.org/en/latest/natspec-format.html#natspec) for all public interfaces (everything in the ABI). I our case that means annotating pretty much everything. After this is implemented, when making changes to contracts we must always take care to update comments/documentation if necessary.

### PR DESCRIPTION
**Pull request by horky2004**

### CODE REVIEW

File: Project/contracts/CustomBallot.sol [hunk 1]
```diff
@@ -83,13 +85,15 @@ contract CustomBallot {
     }
 
     /// @notice Get the voting power of the sender address
+    /// @return votingPower_ voting power
```
Comment (morganjweaver) [inline Project/contracts/CustomBallot.sol:88]: Please add annotation for all return values

**Review by morganjweaver — CHANGES_REQUESTED** [review]
Just the one comment--please add return vals in NatSpec

File: Project/contracts/CustomBallot.sol (hunk 1 above)
Comment (horky2004) [reply Project/contracts/CustomBallot.sol:88]: But they are added!

Comment (morganjweaver) [reply Project/contracts/CustomBallot.sol:88]: oh!  didn't see that.  you're good to go. sorry!

**Pull request merged**